packages/libs/lambda-at-edge/src/types.ts

```diff
@@ -1,4 +1,4 @@
-import {
+import type {
```

Comment:
THis is the change, where this file is moved into `src` and renamed to be a `.ts` extension so it will be included in the dist without updating the `tsconfig.json` etc.